frontend/test/metabase/scenarios/dashboard/chained-filters.cy.spec.js

```diff
@@ -90,4 +190,54 @@ describe("scenarios > dashboard > chained filter", () => {
       });
     });
   }
+
+  context("reproduces metabase#13868", () => {
+    it.only("can use a chained filter with embedded SQL questions", () => {
+      createDashboardWithQuestion({}, dashboardId => {
+        cy.visit(`/dashboard/${dashboardId}`);
+        cy.request("PUT", `/api/dashboard/${dashboardId}`, {
+          embedding_params: {
+            city: "enabled",
+            state: "enabled",
+          },
+          enable_embedding: true,
+        });
+      });
+      // First make sure normal filtering works - we reuse the chained filter test above
+      // Select Alaska as a state. We should see Anchorage as a option but not Anacoco
+      cy.findByText("State").click();
+      popover().within(() => {
+        cy.findByText("AK").click();
+        cy.findByText("Add filter").click();
+      });
+      cy.findByText("City").click();
+      popover().within(() => {
+        cy.findByPlaceholderText("Search by City").type("An");
+        cy.findByText("Anacoco").should("not.exist");
+        cy.findByText("Anchorage").click();
+        cy.findByText("Add filter").click();
+      });
+      cy.contains("Count");
+
+      // Then we make sure it works in embedded mode
+      cy.visit(
+        "/embed/dashboard/eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJyZXNvdXJjZSI6eyJkYXNoYm9hcmQiOjJ9LCJwYXJhbXMiOnt9LCJpYXQiOjE2MDc5NzUwMTMsIl9lbWJlZGRpbmdfcGFyYW1zIjp7InN0YXRlIjoiZW5hYmxlZCIsImNpdHkiOiJlbmFibGVkIn19.nqy_ibysLb6QB9o3loG5SNgOoE5HdexuUjCjA_KS1kM",
+      );
+      cy.findByText("State").click();
+      popover().within(() => {
+        cy.findByText("AK").click();
+        cy.findByText("Add filter").click();
+      });
+      cy.findByText("City").click();
+      popover().within(() => {
+        cy.findByPlaceholderText("Search by City").type("An");
+        cy.findByText("Anacoco").should("not.exist");
+        cy.findByText("Anchorage").click();
+        cy.findByText("Add filter").click();
+
+        cy.contains("Count");
+        cy.findByText("There was a problem").should("not.exist");
```

Comment:
```suggestion
        cy.findByText("There was a problem displaying this chart.").should("not.exist");
```

Either this or regex.

But, this test will fail even when the issue is solved. Unless we wait for this obnoxiously long  xhr
`GET /api/embed/dashboard/eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJyZXNvdXJjZSI6eyJkYXNoYm9hcmQiOjJ9LCJwYXJhbXMiOnt9LCJpYXQiOjE2MDc5NzUwMTMsIl9lbWJlZGRpbmdfcGFyYW1zIjp7InN0YXRlIjoiZW5hYmxlZCIsImNpdHkiOiJlbmFibGVkIn19.nqy_ibysLb6QB9o3loG5SNgOoE5HdexuUjCjA_KS1kM?state=AK&city=Anchorage`
Every `.should("not.exist")` gives us false positive. And every `.should("exist")` or an implicit version of it gives us false negative.

![image](https://user-images.githubusercontent.com/31325167/102140988-6956a280-3e60-11eb-8cee-3be3327726a1.png)

Of course, for the screenshot above the test will fail before this because it can't find "Count" on Y-label, but look at this alternative scenario when I assert that warning icon IS present:
![image](https://user-images.githubusercontent.com/31325167/102142271-678dde80-3e62-11eb-92a5-473c31da4837.png)

Reply:
Yea, that was why I waited on `cy.contains("Count")`, but it's counter-productive because if "Count" exists then by definition ther eis not a problem displaying this chart. :) 

Should I just remove that assertion since I'm waiting for the chart to show up anyway?

Reply:
What I'm saying is that _even when this issue is fixed_ I am afraid it will give us false positive on "Count". Look at the warning icon example.

Reply:
(Conversation from Slack) Realized we had to move the assertions inside the `popover()` code.